bin/hipify-perl

```diff
@@ -26,6 +26,7 @@
 
 #usage hipify-perl [OPTIONS] INPUT_FILE
 
+use warnings;
```

Comment:
`hipify-perl` is autogenerated script. The changes should be done in hipify-clang first: https://github.com/ROCm-Developer-Tools/HIPIFY. There is the comment above:
`#IMPORTANT: Do not change this file manually: it is generated by hipify-clang --perl`

Reply:
thanks!

Reply:
In my defense, I did not change it manually :p

Reply:
That is why there is excessive `use warnings;` in it?

Reply:
I've added those lines `use warnings;` because `/usr/bin/env [command]` only accepts a single executable, not flags :(. Do you see a place where `use warnings;` is repeated / should be dropped?

Reply:
Now I understand: `use warnings;` is added to support `-w`, right?

Reply:
Yes!

Reply:
Ok, I've left hipify-perl as is